SimeonInstaller.ps1

```diff
@@ -900,6 +902,20 @@ CRLFOption=CRLFAlways
 
         Write-Information "Installing Azure DevOps repository and pipelines for '$Name' in project '$Organization\$Project'"
 
+        if (!$PipelineVariables) { $PipelineVariables = @{} }
```

Comment:
why do you need the if/else at all?

Reply:
My understand is that this is an optional parameter, so we initialize it to a blank object when it is not set.

Reply:
Oh are you saying that we can set the default where the parameter is defined?